### ISSUE TITLE
Inefficient imports in python code 

### ISSUE THREAD

**Issue by JollyOmnivore** [ColtonCurrent/BigHornSheepIdentifier#7]
https://github.com/ColtonCurrent/BigHornSheepIdentifier/blob/8672efbba1f7f0dd7e2b3e9f5bbc3f0a6332c9b7/model_cnn.py#L4C1-L8C33

Do this instead :)

from keras.layers import Sequential, Dense, MaxPooling2D, Conv2D, Flatten, Dropout
